SUTrivBot/Models/Commands.cs

```diff
@@ -2,19 +2,114 @@
 using DSharpPlus.CommandsNext.Attributes;
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text;
+using System.Threading.Channels;
 using System.Threading.Tasks;
+using DSharpPlus.Entities;
 
 namespace SUTrivBot.Models
 {
     public class Commands
     {
         // TODO: Add appropriate commands. Current list: AskQuestion, GetPointsList, GetTotalQuestionCount, Echo
 
+        private Dictionary<string, GameState> games;

```

Comment:
private properties should be prefixed with a "_" 

Sorry for being a standards nazi...

Reply:
updating!! and I don't mind at all, I think it's important that the code looks consistent, I just have different habits heh